OpenTrons/opentrons-api
# feat(opentrons-ai-server): AUTH-403 BE chat completion

# Overview

- Extended the code to include storage and tools
- Now server correctly responds to the client

close [AUTH-403](https://opentrons.atlassian.net/browse/AUTH-403)



# Test Plan 
- make unit-test
- make direct-chat-completion
  - when you see shown below, type your prompt
    ` Type a prompt to send to the OpenAI API::` 




# Changelog



# Review requests



# Risk assessment
low



[AUTH-403]: https://opentrons.atlassian.net/browse/AUTH-403?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
